powerform/classes/pf-options.ps1

```diff
@@ -0,0 +1,11 @@
+Class PfOptions{
```

Comment:
i think this is second time i've looked at these guys and each time have to look at it like 5 times to figure out what its doing lol

Reply:
So this is now where the code lives to generate the "ClassPrefix" dynamically.  It adds it as a "member" of the PfOptions, PfDefinition, etc. classes.  Those classes are then inherited by AzBaseResources and the AzContainers which means all classes have a hidden/readonly property of "ClassPrefix" we can use to create new Objects in code at a generic level for access classes, or option classes.